tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -187,19 +184,27 @@ def start_node(self, node):
         self.logger.info(
             f"Starting Open Messaging Benchmark with workers: {worker_nodes}")
 
+        rp_node = self.redpanda.nodes[0]
+        rp_version = "unknown_version"
+        try:
+            rp_version = self.redpanda.get_version(rp_node)
+        except AssertionError:
+            # In some builds (particularly in dev), version string may not be populated
+            pass
+
         start_cmd = f"cd {OpenMessagingBenchmark.OPENMESSAGING_DIR}; \
                     bin/benchmark \
                     --drivers {OpenMessagingBenchmark.DRIVER_FILE} \
                     --workers {worker_nodes} \
-                    --output {self.output_file} \
+                    --output {OpenMessagingBenchmark.RESULT_FILE} \
+                    --service-version {rp_version} \
```

Comment:
Is "unknown_version" a supported value for the `--service-version` flag? Otherwise won't this fail when `redpanda.get_version` fails with AssertionError?

Reply:
Yes. `--service-version` can be any string (no validation). It is just dumped as-is in the OMB result file.  For the context here, I'm doing it because we need to be able to tie an OMB result to a commit/version of redpanda. As we plan to run this on nightly builds in ec2, we should never hit that "unknown_version" issue (they are proper pkg builds).

In dockerized-ducktape tests however, we don't mount the root directory (which has .git metadata) in the test container, so its not possible to get the commit hash.

Reply:
> Yes

Just to be clear, this was the answer to `Is "unknown_version" a supported value for the --service-version flag`
 and not the second half of the question.